consensus/participant.go

```diff
@@ -41,7 +42,7 @@ func NewParticipant(
 	finalized *flow.Header,
 	pending []*flow.Header,
 	options ...Option,
-) (*hotstuff.EventLoop, error) {
+) (module.HotStuff, error) {
```

Comment:
this is one of those cases, where I thought: "Why do we need the `hotstuff.EventLoop` interfaces, when we are essentially not using it and only have a single implementation?"